Paths/bookings

```diff
@@ -1,83 +1,201 @@
-  /bookings/options:
-    get:
-      description: Returns available transport options for given coordinate. Start time can be defined, but is optional. If startTime is not provided, but required by the third party API, a default value of "Date.now()" is used. [from MaaS-API /listing]
+  /planning/availability-check:
+    post:
```

Comment:
Edwin, what's the relationship between   /planning/availability-check  and  /bookings/availability-request  ?  Do they have the same inputs and outputs, other than the fact that one will create a persistent id and the other won't?  Would it be simpler to combine them but keep them separate, e.g. with status=ESTIMATE (no commitment on either side) vs status=PENDING (request-for-quote)?